react/Footer/ToggleContainer/ToggleContainer.js

```diff
@@ -10,7 +10,7 @@ export default function ToggleContainer({ children, secondary, label, name, 'dat
       <input id={name} className={styles.toggle} type="checkbox" data-automation={dataAutomation} />
       <label className={`${styles.toggleLink} ${styles.toggleLabel}`} htmlFor={name}>
         {label}
-        <ChevronIcon direction="down" className={styles.chevron} svgClassName={styles.chevronSvg} />
+        <ChevronIcon direction="down" className={styles.chevron} superstandard />
```

Comment:
Shouldn't this be using the same size as the text rather than `superstandard`? Otherwise it seems like a sign we've got the icon sizing wrong.

Reply:
Happy to go with substandard, however as it's bottom aligned it looks a bit odd due to the icon shape (or maybe it's just me, see screenshot reattached in description). May be it's alright.